### DISCUSSION TITLE
How to install twin in Strapi.

### DISCUSSION THREAD

**Discussion by sondh0127** [ben-rogerson/twin.macro#228] [Q&A] [answered]
I try to integrate twin in Strapi project (https://strapi.io/) by following this guide:
https://github.com/ben-rogerson/twin.examples/tree/master/react-styled-components

But I got an error:

```
 ERROR  Failed to compile with 7 errors                                                                      6:33:32 PM

These dependencies were not found:

* fs in ./node_modules/cosmiconfig/dist/readFile.js, ./node_modules/import-fresh/node_modules/resolve-from/index.js and 4 others
* module in ./node_modules/import-fresh/node_modules/resolve-from/index.js

To install them, you can run: npm install --save fs module
```

After installing the fs and module, It still raises some the error.

`Error: Module not found: Error: Can't resolve 'fs' `

Please give me a suggestion?

**Comment by ben-rogerson** [comment, marked as answer]
I saw the same error in next, [here's how I fixed it](https://github.com/ben-rogerson/twin.examples/blob/master/next-styled-components/next.config.js#L7).
The trick is to to turn 'fs' off in browser environments.
